rust-runtime/aws-smithy-http-server/examples/pokemon-service/tests/simple_integration_test.rs

```diff
@@ -40,6 +40,16 @@ fn get_pokemon_to_capture() -> String {
     pokemons[rand::thread_rng().gen_range(0..pokemons.len())].to_string()
 }
 
+#[tokio::test]
+#[serial]
+async fn test_health_check_operation() {
+    let _program = PokemonService::run();
+    // Give PokémonService some time to start up.
+    time::sleep(Duration::from_millis(500)).await;
```

Comment:
In the Python folder I ended up moving the sleep inside `PokemonService::run()`, just after `Command::spawn()`. This allows you to remove it from all the tests.

Reply:
Ok, I'll do the same for the non-python tests